protostar/utils/starknet_compilation.py

```diff
@@ -67,37 +67,35 @@ def preprocess_contract(
     def compile_preprocessed_contract(
         preprocessed: StarknetPreprocessedProgram,
         add_debug_info: bool = False,
-        is_account_contract: bool = False,
     ) -> ContractClass:
         try:
-            assembled = assemble_starknet_contract(
+            return assemble_starknet_contract(
                 preprocessed_program=preprocessed,
                 main_scope=MAIN_SCOPE,
                 add_debug_info=add_debug_info,
                 file_contents_for_debug_info={},
                 filter_identifiers=False,
-                is_account_contract=is_account_contract,
+                is_account_contract=False,
             )
-            assert isinstance(assembled, ContractClass)
-            return assembled
         except PreprocessorError as err:
-            err_message = err.message
-            if isinstance(err_message, str) and "account_contract" in err_message:
-                raise ProtostarException(
-                    err_message.replace("account_contract", "account-contract")
-                ) from err
+            if isinstance(err.message, str) and "account_contract" in err.message:
```

Comment:
Can we add info about it to documentation? I think it's worth mentioning

Reply:
Should user care about that? What's should be in the docs?

Reply:
That we automatically detect that, users can come from starknet-compiler which requires this flag.

Reply:
I know, but it exists, please provide it as a ::tip or something